### DISCUSSION TITLE
How to have an organised and minimalist feed on GitHub?

### DISCUSSION THREAD

**Discussion by RafaelleDuarte** [community/community#23523] [New to GitHub]
<p>Yesterday I posted a game on the repositories section of GitHub, however I am totally new to GitHub and I don’t know how to mould my feed to be organised and minimalist.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you may want to check this out</p>
<p><a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/about-your-personal-dashboard">https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/about-your-personal-dashboard</a></p>
